src/main/com/github/soerxpso/portalfix/PortalFix.java

```diff
@@ -98,11 +97,15 @@ public void run() {
 	class TeleportIfStill extends BukkitRunnable implements Listener {
 		private UUID p;
 		private Location l;
+		private Countdown countdown;
 		
 		public TeleportIfStill(UUID p, Location l) {
 			this.p = p;
 			this.l = l;
 			getServer().getPluginManager().registerEvents(this, PortalFix.getPlugin());
+			countdown = new Countdown((Config.getWaitTime() - Config.getCountdownInterval()) * 20, 
+					Config.getCountdownInterval() * 20, getPlugin().getServer().getPlayer(p));
+			countdown.runTaskLater(PortalFix.getPlugin(), Config.getCountdownInterval() * 20);
```

Comment:
IIRC you should be able to use runTaskTimer() here; just keep your count of runs, and have it cancel itself when you've notified the user enough times.